### ISSUE TITLE
[Spell] Chimera Shot 

### ISSUE THREAD

**Issue by crulinjr** [WoWManiaUK/Redemption#2770]
How it works atm:

So the Chimera Shot - Serpent Sting part of the spell is now working as intended but is not doing crits anymore. 

![wowscrnshot_100218_073252](https://user-images.githubusercontent.com/40045543/46348930-3d033280-c616-11e8-8d19-48bda8b882ed.jpg)

How it should work:
The Chimera Shot - Serpent Sting crit damage should be modified by the Mortal Shots talent and the meta gem.
avr normal hit dmg * 2.3 * 1.03 = avg normal hit dmg * 2.369

**Comment by wow-mania** [comment]
None of the work done should have changed this. Also, probabilities are not worked out like this.

**Comment by crulinjr** [comment]
ty :D

**Comment by crulinjr** [comment]
Closed? The bug still persists

**Comment by Dranah** [comment]
Confirmed, ability is no longer possible to crit with.
This is unintended, proof: http://www.worldoflogs.com/reports/rt-fnpskaevlhr3lp63/details/28/?s=4108&e=4275
This is a log from retail 3.3.5a with a hunter's chimera-serpent showing hits of average 3358 and crits of average 8726. So the ability should be able to crit and even receives the same critical damage bonus of ranged shots as Chimera shot itself.

**Comment by crulinjr** [comment]
i love you dranah

**Comment by Keader** [comment]
wtf?
Damage is based in other spell, and can crit? lul

**Comment by Keader** [comment]
Test now.
Maybe damage will not appear as critical, but will be a high value.

**Comment by Dranah** [comment]
Yes it can crit, just like priests Improved Devouring Plague. It's not that absurd.

PS: There is no change

**Comment by Keader** [comment]
was not in ptr yet haha

**Comment by Dranah** [comment]
Chimera - Serpent now is capable of doing criticals, though it is not displayed as a critical visually (just does double the damage).
Problem is this critical damage seems unaffected by critical damage bonus mods (talents such as Mortal Shots, Marked for Death and the meta), it deals a fix 200% of spell damage, and this is not intended.

**Comment by Dranah** [comment]
you can pm me on discord if you want to discuss anything @crulinjr

**Comment by Dranah** [comment]
Chimera-serpent is now able to crit and will also receive proper critical damage bonus from talents/meta.
Note the visual display cannot be fixed, these crits will be shown as hits too, on both logs and recount.
